gpkit/constraints/model.py

```diff
@@ -174,7 +178,7 @@ def debug(self, solver=None, verbosity=1, **solveargs):
                                       constsrelaxed.origvars,
                                       min_return=0 if sol["boundedness"] else 1)
                 if relaxed:
-                    if sol["boundedness"]:
+                    if sol.get("boundedness", None):
```

Comment:
sorry what does this line do?

Reply:
if there's no boundedness element (because process_results wasn't run) then `sol.get("boundedness", None)` will return None instead of a KeyError

Reply:
got it

Reply:
Removed this; debug now automatically calls just the process_result of Bounded and skips all other process_result calls